src/test/compile-fail/discrim-overflow-2.rs

```diff
@@ -24,7 +24,7 @@ fn f_i8() {
     enum A {
         Ok = i8::MAX - 1,
         Ok2,
-        OhNo, //~ ERROR enum discriminant overflowed on value after 127: i8; set explicitly via OhNo = -128 if that is desired outcome
+        OhNo, //~ ERROR enum discriminant overflowed on value after 127i8; set explicitly via OhNo = -128i8 if that is desired outcome
```

Comment:
Nit: I wish we would pretty print this as `127_i8`, so much more readable. I'd personally like that to be an official style recommendation.